utils/hwinfo/Makefile

```diff
@@ -0,0 +1,79 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=hwinfo
+PKG_VERSION:=21.70
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/openSUSE/hwinfo/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=bc3c4a4498e4abc32a54497ced715bbae5dfd19dd999da294bca6d69fea2db52
+
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_MAINTAINER:=Alberto Bursi <bobafetthotmail@gmail.com>
+
+PKG_BUILD_DEPENDS:= hwinfo/host
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
+
+# Execute in host build directory
+HOST_MAKE_FLAGS += -C $(HOST_BUILD_DIR)
+
+define Host/Configure
+	# copy uuid.h to another location in host build dir as that's where this package expects it
+	$(INSTALL_DIR) $(STAGING_DIR_HOST)/include/uuid/
+	$(CP) $(STAGING_DIR_HOST)/include/e2fsprogs/uuid/uuid.h $(STAGING_DIR_HOST)/include/uuid/uuid.h
+endef
+
+define Host/Compile
+	# Build using host compiler and let it generate the files we need
+	# CFLAGS, CPPFLAGS & LDFLAGS need to be passed with CC because they are being ingored
+	CC="$(HOSTCC) $(HOST_CFLAGS) $(HOST_CPPFLAGS) $(HOST_LDFLAGS)" $(MAKE) $(HOST_MAKE_FLAGS)
+endef
+
+define Build/Configure
+	# Fetch prebuilt files from host build dir
+	$(INSTALL_DIR) $(PKG_BUILD_DIR)/src/isdn/cdb/
+	$(CP) $(HOST_BUILD_DIR)/src/isdn/cdb/isdn_cdb $(PKG_BUILD_DIR)/src/isdn/cdb/
```

Comment:
I think you have to install this and then use it from STAGING_DIR, I remember that I encountered this problem on build bots if I used directly the HOST_BUILD_DIR

Reply:
yes, this is the problem
`cp: cannot stat '/home/build/build_dir/build_dir/hostpkg/hwinfo-21.70/src/isdn/cdb/isdn_cdb': No such file or directory
`

Reply:
seems to be working, thanks